app/models/task.rb

```diff
@@ -0,0 +1,60 @@
+class Task < ActiveRecord::Base
+  belongs_to :event
+  validates :event, presence: true
+  validates :title, presence: true
+  validates :start_time, presence: true
+  validates :end_time, presence: true
+  validates_chronology :start_time, :end_time
+
+  # has_many :requirements
+  # has_many :assignments, through: :requirements
+
+  STATUS_CHOICES_ARRAY = ['Empty', 'Partially Filled', 'Full', 'Cancelled']
+
+  # The following provides the equivalent of:
+  # STATUS_CHOICES = { 'Empty' => 0, 'Partially Filled' => 1, 'Full' => 2, 'Cancelled' => 3 }
+  # This is used to give a sortable numeric value based on the order of the above array.
+  STATUS_CHOICES = STATUS_CHOICES_ARRAY.map.with_index { |v, i| [v, i] }.to_h
+
+  def status
```

Comment:
To be clear to others who may be watching, this is only for testing. As we build out the other models needed for tracking events, this will go away.